src/app/pages/search/challenge-search/challenge-search.component.ts

```diff
@@ -54,6 +57,8 @@ const defaultChallengeSearchQuery: ChallengeSearchQuery = {
 export class ChallengeSearchComponent implements OnInit, AfterViewInit {
```

Comment:
@rrchai Must implement `OnDestroy` because the component includes `ngOnDestroy()`.

Reply:
The PR can then be merged.